odometry_mode: 1  # LIO = 0, LICO = 1

log_path: /home/jerry/log
config_path: /home/jerry/catkin_ws/src/Coco-LIC/config
lidar_yaml: /viral/lidar.yaml
imu_yaml: /viral/imu.yaml
camera_yaml: /viral/camera.yaml

bag_path: /home/jerry/dataset/eee_01.bag  # eee_xx.bag、nya_xx.bag、sbs_xx.bag
bag_start: 0.
bag_durr: -1


### SplineParams ###
t_add: 0.1  # [s] trajectory is updated every delta_t seconds, no adjustment needed
distance0: 0.03  # [s] the time duration corresponding to the first 4 control points, no adjustment needed
non_uniform: true  # true for non-uniform b-spline, false for uniform b-spline
division_coarse: 1  # if it is uniform, decide how many control points to place in every delta_t seconds, must be a non-zero integer


### OptimizationParams ###
t_begin_add_cam: 5.0  # [s] visual constraints will be added after the system has been successfully initialized for a while
lidar_iter: 2
use_lidar_scale: true  # scale the lidar weight
lidar_weight: 500.0
image_weight: 4.0
# imu：VectorNav VN100（385Hz）
# derived from data manual https://www.vectornav.com/products/detail/vn-100
gyroscope_noise_density: 3.67396706572e-03
gyroscope_random_walk: 2.66e-05                    
accelerometer_noise_density: 3.65432018302e-02
accelerometer_random_walk: 4.33e-04         


### EvalutaionParams ###
is_evo_viral: true 